### PR TITLE
feat: Add explicit pagination with 5 sightings/page

### PR DESCRIPTION
**Pull request by mcraig567**
Number of sightings can be adjusted by showLimit in sightingsController.js

### CODE REVIEW

File: client/src/index.jsx [hunk 1]
```diff
@@ -173,7 +191,43 @@ class App extends React.Component {
...
+		this.setState({
+			page: this.state.page += 1
+		});
+		this.viewHomepage();
+	}
+
 	render() {
+		let backButton;
+		if (this.state.page === 0) {
+			backButton = null;
```
Comment (DukeBaird) [inline client/src/index.jsx]: Stylistic suggestion: don't make the buttons go away, just make the disabled. its jarring to have the screen change and buttons move around

File: server/lib/sightingsController.js [hunk 2]
```diff
@@ -11,8 +11,16 @@ const logger = require('./logger.js');
 // Get all sightings in the database
 exports.getSightings = (searchOpts = {}) => {
 	logger.info('Getting Sightings...');
-	logger.info(`Using some variables so lint doesn't scream: ${searchOpts}`);
-	return Sighting.find({});
+	const showLimit = 5;
+	let skips = 0;
+
+	if (searchOpts.page) {
```
Comment (DukeBaird) [inline server/lib/sightingsController.js:17]: This will need to get moved into query builder & tested ofc

**Review by DukeBaird — APPROVED** [review]
lgtm

File: server/lib/sightingsController.js (hunk 2 above)
Comment (mcraig567) [reply server/lib/sightingsController.js:17]: Would you normally merge as is, and then in the profile branch update it? Or update it now before merge

**Pull request merged**